lib/gems/pending/appliance_console/external_httpd_configuration.rb

```diff
@@ -15,6 +15,7 @@ module ExternalHttpdConfiguration
       SSSD_CONFIG          = "/etc/sssd/sssd.conf".freeze
       PAM_CONFIG           = "/etc/pam.d/httpd-auth".freeze
       HTTP_KEYTAB          = "/etc/http.keytab".freeze
+      HTTP_CONFIG_FILE     = "/etc/httpd/conf.d/manageiq-redirects-ws".freeze
```

Comment:
@carbonin If this is not yet created restarting httpd by the appliance_console/`Configure External Auth` fails. This file is created when manageIQ is configured. It is possible that the file was created but someone could have stopped the httpd service.

I had discussed this very scenario with @abellotti and we felt the safest approach would be to check for the file.